core-profile/core-tck/pom.xml

```diff
@@ -132,6 +132,7 @@
                             <target>
                                 <fileset id="jakarta-api-jars"
                                          dir="${jboss.home}${file.separator}modules/system/layers/base/jakarta/">
+                                    <include name="**/activation/api/main/*.jar"/>
```

Comment:
What's this for?

Reply:
I can try running without this, but the REST Client requires `jakarta.activation.DataSource`.

Reply:
This is not needed, I'll remove it. Not sure why I added it. Good catch!